examples/flux-todomvc/js/components/TodoApp.react.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2014-2015, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+/**
+ * This component operates as a "Controller-View".  It listens for changes in
+ * the TodoStore and passes the new data to its children.
+ */
+
+var Footer = require('./Footer.react');
+var Header = require('./Header.react');
+var MainSection = require('./MainSection.react');
+var React = require('react');
+var TodoStore = require('../stores/TodoStore');
+var ObservableStateMixin = require('thundercats').ObservableStateMixin;
+
+/**
+ * Retrieve the current TODO data from the TodoStore
+ */
+function getTodoState(state) {
+  var areAllComplete = true;
```

Comment:
@morenoh149 I would probably change the name of this function or make it an anon function and put it directly in the map function. 'getTodoState' makes it seem like the function is a getter instead of just a mapping function.